megamek/src/megamek/common/Game.java

```diff
@@ -3651,4 +3659,17 @@ private void checkPositionCacheConsistency() {
         }
     }
 
+    /**
+     * Get a string representation of the UUId for this game.
+     *
+     * @return
+     */
+    public String getUUIDString() {
+        if (uuid == null) {
```

Comment:
Curious: when would `uuid` be null?

Reply:
It wouldn't be.  As the code is written now, it can't be (the constructor initializes it and there's no setter).  I'm pretty sure I wrote this just to be defensive.

Actually, I think I was thinking about saved games from previous versions, but those might just fail to deserialize.

It's easy enough to remove the condition if you want, but I also don't really think it harms anything.  Maybe it would be better to log a message at that point, since it's a condition we'd expect not to happen.

Reply:
I'm okay as is. We also don't guarantee save games across versions so no worries there.